CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.4 (2020-05-06)
+
+- Fixed the column selector action to properly display in the correct position inside a list toolbar. [#67](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/67)
```

Comment:
We don't really need "properly." I'd delete it, but I'll approve the PR since it's not a big deal if you don't want to make the change.

Reply:
@blackbaud-johnly I'm not going to push this until tomorrow so I made the change :)